ci-operator/step-registry/shiftstack/cleanup/shiftstack-cleanup-ref.yaml

```diff
@@ -6,9 +6,17 @@ ref:
     requests:
       cpu: 1000m
       memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: shiftstack-slack-hooks
+    mount_path: /var/run/slack-hooks
   env:
   - name: OS_CLOUD
     default: openstack
+  - name: CLUSTER_TYPE_OVERRIDE
```

Comment:
I just asked this, let me cite Pierre:

> Doesn't say the cluster

Reply:
In its message to Slack, Prune should also say which cluster the resources are from. To do that, [it uses the `CLUSTER_TYPE` environment variable](https://github.com/shiftstack/prune/blob/25fee754317ecbe044e719303a3d8d8901582065/slack.go#L15-L17), which is the standard way to identify a cluster in Prow; except [we use an override trick](https://github.com/openshift/release/blob/d60749393ebe3054b573706090dbfc4f68f01224/ci-operator/config/shiftstack/prune/shiftstack-prune-main.yaml#L46) in the cleanup config to avoid consuming Boskos leases.

That's why we need the `CLUSTER_TYPE_OVERRIDE` variable in this script, and we use it a bit like we do in [openstack-conf-clouds](https://github.com/openshift/release/blob/d60749393ebe3054b573706090dbfc4f68f01224/ci-operator/step-registry/openstack/conf/clouds/openstack-conf-clouds-commands.sh#L11).